libtsuba/include/katana/TxnContext.h

```diff
@@ -85,9 +92,11 @@ class KATANA_EXPORT TxnContext {
 
   bool GetTopologyWrite() const { return topology_write_; }
 
+  bool GetCommitManifest() const { return commit_manifest_; }
+
 private:
-  std::string ConcatRDGProperty(std::string rdg_dir, std::string prop) {
-    return rdg_dir + kPropSeparator + prop;
+  inline std::string ConcatRDGProperty(std::string rdg_dir, std::string prop) {
+    return Uri::JoinPath(rdg_dir, prop);
   }
 
   static constexpr char kPropSeparator[] = "/propertyseparator/";
```

Comment:
Just curious, why is this a character array?

Reply:
I'm removing this since it's unused. It was a character array since `static` member variables with initialization on declaration only works with `char[]`. I learnt that it only works for `std::string` after c++17. I got a building error with our settings when I used `static constexpr std::string`.